### PR TITLE
Translation into German

### PR DESCRIPTION
**Pull request by finben**
I sent a copy to Jonathen Gil to proofread by translations. But here is what I came up with so far. One todo is that all the titles of the Bible books need to be added to our localized strings. Everything else is looking good though in my simulator for German.

### CODE REVIEW

File: Hymns/Browse/BrowseTab.swift [hunk 1]
```diff
@@ -16,19 +16,19 @@ extension BrowseTab {
     var label: String {
         switch self {
         case .tags:
-            return "Tags"
+        return NSLocalizedString("Tags", comment: "")
```
Comment (skywalkerdude) [inline Hymns/Browse/BrowseTab.swift:19]: What is the "comment" used for?

Comment (skywalkerdude) [reply Hymns/Browse/BrowseTab.swift:19]: Ah, I see that it'll get auto-ported to the Localizable file. Can we go through and add them in to make it easy to translate? 😛 I'll help with it too when I get time  https://developer.apple.com/documentation/foundation/nslocalizedstring

Comment (finben) [reply Hymns/Browse/BrowseTab.swift]: Do you mean add the same string to the comment? If that is what you mean I did that below.

**Comment by skywalkerdude** [comment]
@finben I fixed it up a little bit and added some new strings that we added since you last committed. Could you translate those also, then submit?

**Pull request merged**